### ISSUE TITLE
Performance is worse while training

### ISSUE THREAD

**Issue by 186big** [MaxLeo-LMZ/BEVFusion_Learningnote#5]
Thank u for your great job.
I use your code and pretrained model to train a Lidar + Camera detection model in Nuscene dataset , but the more epochs I trained, the worse it got.
The loss is lower But the map is lower too.
![image](https://github.com/MaxLeo-LMZ/BEVFusion_Learningnote/assets/97373109/01f7765e-5e8d-4e32-ae73-8ac55c783165)
